omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -16,6 +16,12 @@
       ssl_protocols <%= @ssl_protocols %>;
       ssl_ciphers <%= @ssl_ciphers %>;
       ssl_prefer_server_ciphers on;
+
+      <% if node['private_chef']['nginx']['ssl_client_ca'] -%>
+        ssl_verify_client on;
+        ssl_client_certificate <%= node['private_chef']['nginx']['ssl_client_ca'] %>
+        ssl_verify_depth 10;
```

Comment:
Would we have reason to want to expose this an attribute?

Reply:
Changing the depth to 2. 10 is much too permissive.